### DISCUSSION TITLE
Expose all the emojis as reactions

### DISCUSSION THREAD

**Discussion by cilerler** [community/community#3082] [Discussions]
Would you please remove the reaction emoji restriction?  

I see myself using ☠️ and 🤗 a lot, and unfortunately, those are not on the list.  I get the reasoning behind the restriction, but if that is something you restrict, at least allow admins to pick which emojis are permitted.

_P.S. I would also like to see this feature in the issues._

### Actual
![image](https://user-images.githubusercontent.com/385958/103355101-6b814980-4a7b-11eb-883e-a006affe369b.png)

### Expected
![image](https://user-images.githubusercontent.com/385958/103355438-3e816680-4a7c-11eb-973f-c26d089d2c6e.png)

**Comment by PranamBhat** [comment]
Hello!!

"**Reactions**" should be always a small list. There are only few emoji's we can expect over reaction window. Whatever you are expecting is - comes under comment/reply section. That is not a reaction window!! 

However for comment section, we can always use Windows logo key   + . (period). 

[https://support.microsoft.com/en-us/windows/windows-10-keyboard-tips-and-tricks-588e0b72-0fff-6d3f-aeee-6e5116097942#:~:text=To%20use%20it%3A,emojis%20for%20one%20you%20like.](https://support.microsoft.com/en-us/windows/windows-10-keyboard-tips-and-tricks-588e0b72-0fff-6d3f-aeee-6e5116097942#:~:text=To%20use%20it%3A,emojis%20for%20one%20you%20like.)

**Reply by cilerler** [reply]
As I stated in my initial comment, _"I get the reasoning behind the restriction, but if that is something you restrict, at least allow admins to pick which emojis are permitted"_ like labels. So comments are outside of the scope since we do have access to all emojis.

**Reply by seaneez** [reply]
I agree. Why don't you allow each team to make its own emoji lists?

**Reply by milstan** [reply]
I bet that if you run an anlysis to see how much which emoji reaction is used, you're likely to see the existing fixed list contains some rarely used ones. In my team I've only seen 👍 and ❤️ being used. On other tools offering more choices, 🙏 and ✅ are used almost as often by this same team.

If you keep the restriction of number of options, but allow changes to the actual emojis, you'll make work with github even more eficient.

**Reply by tmccombs** [reply]
Using a comment is not a good workaround, because that notifies everyone who is subscribed to the PR/Issue.

**Comment by lavie** [comment]
The limitation on reaction emojis is very weird. At the very least it should be expanded substantially. Though I don't really see why not just allow all emojis. Is it a UI thing? Wanting to avoid reaction spam? Can always tuck the overflowing reactions into some triple-dot area...

**Reply by abinoda** [reply]
I think the limitation makes sense because the constraint converges the discussion around a smaller set of emoji instead of leading to emoji spam.

**Reply by cilerler** [reply]
> _"I get the reasoning behind the restriction, but if that is something you restrict, at least allow admins to pick which emojis are permitted" like labels_

**Reply by tmccombs** [reply]
> I think the limitation makes sense because the constraint converges the discussion around a smaller set of emoji

However, if the set is too limited, it leads to confusion as a single emoji is used for multliple purposes. 

For example, I've seen confusin about the following existing reactions:

- :rocket: , some people use it to mean "ship it/launch it". Others use it to mean "upvote" or "I agree", since there are not any other emoji that obviously fit that, and :rocket: doesn't have a clear meaning. While these are all positive responses, there is nuance between them. Also, I've seen people just unsure what is meant at all by the :rocket: emoji.
- :eyes: is used for "I'll look into this", "I've seen this", "this looks interesting", or "I'm watching this Issue/PR/discussion". It is often hard to tell which is intended from context, and these are pretty significant differences.
- :+1:  is used to mean, "+1"/upvote/like, I've seen and acknowledged your comment, and "I agree". Like :rocket:, the meanings are generally similar, and positive, but they aren't quite the same.
- :-1: is used to mean "-1/downvote/dislike", "I disagree", "this is wrong", and I'm sad about this
- :laughing: is used to me "I'm happy about this", "this is funny", and "sarcasm"
- :confused: is used to mean "I'm confused" and "I'm unhappy about this" 

There also isn't a clear reaction to use for things like "this is done", "oops, I made a mistake", or "I'll look into this/do this/assign this to myself".

I think that if there is  small fixed set of emoji, then it should probably be expanded somewhat. 

Also, I don't think it would be a bad thing to have some kind of guide or cheatsheet on what meaning those reactions are intended to convey.

**Comment by markseuffert** [comment]
Discord allows any emoji as reaction, would be nice to have something similar in GitHub discussions.

**Comment by kj800x** [comment]
Some of the emoji names in the current set are also highly misleading at times. I have seen :confused: be used to express bittersweet emotions and sorrow such as when a large project is finally sunset. I've also seen :laugh: be used to express joy, which is very different from laughter. Having a PR that makes someone happy is very different from someone laughing at my PR. With the recent push for more communication to be done online, it's more important than ever that people don't misunderstand, and emojis are rife for misunderstanding especially when limited so much.

**Comment by kenwang-coursera** [comment]
+1

I just want a "+" emoji... a thumbs up does _not_ mean the same thing as "+". Allowing admins to configure their own is super powerful when the org has their own standards of what each emoji represents.

**Comment by jameshiltonncino** [comment]
I would love to express my emotional reaction to some of these comments but I am hindered by a limited selection of emoji reactions

**Comment by serinamarie** [comment]
I personally would love to see a 😮  or a 🤯 , when I've learned something new on a PR :)
